Makefile.dep

```diff
@@ -81,7 +81,11 @@ ifneq (,$(filter tlsf-malloc,$(USEMODULE)))
 endif
 
 ifneq (,$(filter tinydtls_sock_dtls, $(USEMODULE)))
-    USEPKG += tinydtls
+  USEPKG += tinydtls
```

Comment:
Unrelated

Reply:
Doesn't it make sense to fix the whitespace when touching that file?